pkg/util/api/response.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+)
+
+type Response struct {
+	Status    string       `json:"status"`
+	Data      interface{}  `json:"data,omitempty"`
```

Comment:
This could unexpected break some consumer code because now all 5xx responses will not have a `data` key. Maybe we can move this on a separate PR?

Reply:
Thanks for the comment. Sure I will remove this change.

Reply:
Fixed in latest commit